src/Misc/layoutbin/powershell/Add-MavenCapabilities.ps1

```diff
@@ -1,10 +1,40 @@
 [CmdletBinding()]
 param()
 
+function Get-MavenDirectoryFromPath {
```

Comment:
Get-Command mvn should get you the location without all this parsing. Please try it out.

Reply:
There is also a module you can use: https://github.com/Microsoft/vsts-agent/blob/f7e9a7ee42d0c30f0d31d449bb51d95bd6e9e0fb/src/Misc/layoutbin/powershell/CapabilityHelpers/CapabilityFunctions.ps1#L1